csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,667 @@ local commonlib = import 'common-lib/common/main.libsonnet';
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableSentBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableReceivedBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableReadBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableWriteBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+        ]
+      )
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Time',
+          },
+          properties: [
+            {
+              id: 'custom.hidden',
+              value: true,
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'instance_name',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Instance',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'job',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Job',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'project_id',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Project ID',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableCpuUtilization',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Cpu utilization',
+            },
+            {
+              id: 'unit',
+              value: 'percent',
+            },
+            {
+              id: 'custom.width',
+              value: 100,
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                mode: 'basic',
+                type: 'gauge',
+                valueDisplayMode: 'text',
+              },
+            },
+            {
+              id: 'thresholds',
+              value: {
+                mode: 'absolute',
+                steps: [
+                  {
+                    color: 'yellow',
+                    value: null,
+                  },
+                  {
+                    color: 'green',
+                    value: 30,
+                  },
+                  {
+                    color: 'red',
+                    value: 85,
+                  },
+                ],
```

Comment:
Can we sync on the threholds we use? I am using
- Base Green 
- from 70 Orange
- from 90 Red

Let's decide together which one to use in all our dashboards. What do you think?

Reply:
I got these thresholds from Vasil, so perhaps you can update yours?

Reply:
but why not use like 70 for yellow or orange for example?

Reply:
I believe the ideal CPU value is between 30 and 85 (green) because it should not be under utilized, or over utilized